redwood-lazylayout-uiview/src/commonMain/kotlin/app/cash/redwood/lazylayout/uiview/UIViewLazyList.kt

```diff
@@ -273,6 +272,7 @@ internal class LazyListContainerCell(
 
   override fun prepareForReuse() {
     super.prepareForReuse()
+    removeAllSubviews()
```

Comment:
this sounds right for what we have. wishful thinking would be that these views stay put and redwood could just modify the values of them to actually reuse the cell